server/athenian/api/models/state/versions/e7afe365ab0e_create_repository_sets_table.py

```diff
@@ -0,0 +1,42 @@
+"""Create repository_sets table
+
+Revision ID: e7afe365ab0e
+Revises: 34eafe9e7cd9
+Create Date: 2020-01-05 07:46:09.270335+00:00
+
+"""
+from datetime import datetime
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e7afe365ab0e"
+down_revision = "34eafe9e7cd9"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    def count_items(ctx):
```

Comment:
So adding an inner `Context` is a standard that is handled by sqlalchemy and alembic theirselves?

Reply:
Yes, this is how the ORM handles them. Strictly speaking, alembic does not care about defaults and onupdate-s, so I should remove them from here.